.github/workflows/image.yaml

```diff
@@ -22,7 +26,7 @@ jobs:
       - name: Install Cosign
         uses: sigstore/cosign-installer@main
         with:
-          cosign-release: 'v1.2.1'
+          cosign-release: 'v1.4.0'
```

Comment:
A lot of bugs were fixed in 1.4.0 (nothing that would affect this usage AFAIK), so it may be worth picking up 1.4.1.

Otherwise this LGTM!

Reply:
thanks @mattmoor 

all green now :) https://github.com/kyverno/kyverno/actions/runs/1664576388